scripts/map.js

```diff
@@ -6,11 +6,11 @@ var dataLayer // GeoJSON layer with district data
 var overlays = {} // An object to contain overlay layer groups, eg `transit`
 
 var zone2color = {
-  R: '#645097', // primarily residential, satisfied
-  M: '#815196', // mixed with residential, satisfied
-  N: '#BA6CA4', // nonresidential, satisfied
-  NS: '#d0d0d0', // not satisfied
-}
+  R: "#BB22CA", // primarily residential, satisfied
+  M: "rgba(113, 70, 116, 0.92)", // mixed with residential, satisfied
```

Comment:
```suggestion
  M: "#714674eb", // mixed with residential, satisfied
```
Tip: You can use 8 character Hex codes (like #11223344) for transparent RGBA colors

Reply:
Oh! I didn't know the longer hex codes altered the opacity! That's good to know!